client/src/app/users/add-user.component.html

```diff
@@ -1,66 +1,68 @@
-            <mat-hint>Not required in Angular for demonstration, still required by server</mat-hint>
-          </mat-form-field>
+            <mat-form-field>
+              <mat-label>Company</mat-label>
+              <input matInput placeholder="Company"
+                    formControlName="company">
+              <mat-hint>Not required in Angular for demonstration, still required by server</mat-hint>
+            </mat-form-field>
 
-          <mat-form-field>
-            <mat-label>Email</mat-label>
-            <input matInput placeholder="Email"
-                  formControlName="email" required>
-            <mat-error *ngFor="let validation of addUserValidationMessages.email">
-              <mat-error class="error-message" data-test='emailError'
-                        *ngIf="addUserForm.get('email').hasError(validation.type) && (addUserForm.get('email').dirty || addUserForm.get('email').touched)">
-                {{validation.message}}
+            <mat-form-field>
+              <mat-label>Email</mat-label>
+              <input matInput placeholder="Email"
+                    formControlName="email" required>
+              <mat-error *ngFor="let validation of addUserValidationMessages.email">
+                <mat-error class="error-message" data-test='emailError'
+                          *ngIf="addUserForm.get('email').hasError(validation.type) && (addUserForm.get('email').dirty || addUserForm.get('email').touched)">
+                  {{validation.message}}
+                </mat-error>
               </mat-error>
-            </mat-error>
-          </mat-form-field>
+            </mat-form-field>
 
-          <mat-form-field>
-            <mat-label>Role</mat-label>
-            <mat-select required formControlName="role">
-              <mat-option value="admin">Admin</mat-option>
-              <mat-option value="editor">Editor</mat-option>
-              <mat-option value="viewer">Viewer</mat-option>
-            </mat-select>
-            <mat-error *ngIf="addUserForm.get('role').hasError('required')">You must make a selection</mat-error>
-            <mat-error *ngIf="addUserForm.get('role').hasError('pattern') && !addUserForm.get('role').hasError('required')">
-              Your selection is invalid
-            </mat-error>
-          </mat-form-field>
+            <mat-form-field>
+              <mat-label>Role</mat-label>
+              <mat-select required formControlName="role">
+                <mat-option value="admin">Admin</mat-option>
+                <mat-option value="editor">Editor</mat-option>
+                <mat-option value="viewer">Viewer</mat-option>
+              </mat-select>
+              <mat-error *ngIf="addUserForm.get('role').hasError('required')">You must make a selection</mat-error>
+              <mat-error *ngIf="addUserForm.get('role').hasError('pattern') && !addUserForm.get('role').hasError('required')">
+                Your selection is invalid
```

Comment:
Do we want a more specific error message here? I know this isn't a new change, but it does seem a little bit vague.